### DISCUSSION TITLE
Best way to convert UHD footage to SDR lossless?

### DISCUSSION THREAD

**Discussion by bstonearch** [kmcclive/tricycle#130] [Q&A]
Hello, I have successfully used Tricycle to convert my 4k UHD file to 4k SDR. However, my SDR file size is significantly smaller. The original UHD file size is around 78GB whereas the converted SDR file size is around 7GB. Are there certain settings that I need to change in order to maintain a lossless conversion? 

For format, should I be using HEVC or AVC?

In "tools > options > advanced" should I be changing my x264 or x256 presets to very slow or placebo?

In "tools > options > video" should I be changing the Quality settings, number of steps, etc?

Also, a second question; When converting a 10bit HDR file to SDR, is it converting the file to 10 bit SDR or 8 bit SDR?

Thanks in advance!

**Comment by docgonzo2015** [comment]
x264 & x265 aren't lossless conversion codecs at all.

**Comment by kmcclive** [comment]
Hi, @bstonearch.  Sorry for the very late reply.  I guess I don't get notified when someone starts a discussion.

As @docgonzo2015 mentioned, HEVC and AVC aren't lossless codecs.  The way that Tricycle tunes them is to use CRF, which means that it is targeting a specific quality, not a bitrate.  You can decrease the value for the "Best" quality setting from 18 to something like 16, but I chose 18 specifically because I find it difficult to discern any difference in quality beyond that.  As to whether you should use HEVC or AVC, that depends on what you will be playing the videos on.  HEVC supports higher compression (without sacrificing quality), but AVC is supported on more players/devices.

To answer your last question, when converting HDR to SDR, the video produced will be 8-bit, not 10-bit.  10-bit video, along with mastering metadata, is a component of HDR.  Keeping the video at 10-bit wouldn't really be SDR.
